components/dashboard/tailwind.config.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
                 'gitpod-kumquat-light': '#FFE4BC',
                 'gitpod-kumquat': '#FFB45B',
                 'gitpod-kumquat-dark': '#FF8A00',
+                'gitpod-kumquat-darker': '#f28300',
```

Comment:
**thought**: Oh, [way down we go](https://youtu.be/0-7IHOXkiV8)! 😸

Reply:
just to give it a little hover effect. the normal kumquat felt too bright for the buttons.😇